### PR TITLE
Update initializers.py

### PR DESCRIPTION
**Pull request by PariksheetPinjari909**
Added missing documentation string

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet, @pavithrasv: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

File: tensorflow/python/keras/initializers.py [hunk 1]
```diff
@@ -84,6 +84,9 @@ class TruncatedNormal(TFTruncatedNormal):
     seed: A Python integer. Used to create random seeds. See
       `tf.set_random_seed` for behavior.
     dtype: The data type. Only floating point types are supported.
+
+  Returns:
+      TruncatedNormal instance.
```
Comment (penpornk) [inline tensorflow/python/keras/initializers.py]: The indentation is a bit off.
```suggestion
    A TruncatedNormal instance.
```

File: tensorflow/python/keras/initializers.py [hunk 2]
```diff
@@ -105,6 +108,9 @@ class RandomUniform(TFRandomUniform):
     seed: A Python integer. Used to create random seeds. See
       `tf.set_random_seed` for behavior.
     dtype: The data type.
+
+  Returns:
+      RandomUniform instance.
```
Comment (penpornk) [inline tensorflow/python/keras/initializers.py]:
```suggestion
    A RandomUniform instance.
```

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for your PR and sorry for our delay!

**Comment by PariksheetPinjari909** [comment]
@penpornk I have accepted your suggestion, can you do cla check. Thanks

**Review by penpornk — APPROVED** [review]
Thank you again for your contribution!

CLA: The other contributor is me through github UI so it doesn't detect my CLA. I have verified that the CLA is okay. To make it technically correct, I'll also say I'm okay with contributing to this PR.

**Comment by penpornk** [comment]
@PariksheetPinjari909 Could you please resolve the conflict?

**Comment by PariksheetPinjari909** [comment]
I am closing the PR as the changes are already taken.
